src/components/Author.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  import Avatar from "@app/components/Avatar.svelte";
+  import { formatNodeId } from "@app/lib/utils";
+
+  export let nodeId: string;
+  export let noAvatar: boolean = false;
```

Comment:
Doesn't look like we're using this prop anywhere in the callsites. Can we drop it?